.github/actions/setup-javascript/action.yml

```diff
@@ -1,15 +1,14 @@
 name: Set up Javascript
-description: Installs Node.js and dependencies defined in package.json
+description: Installs Bun.js and dependencies defined in package.json
 runs:
   using: composite
   steps:
-    - name: Set up node
-      uses: actions/setup-node@ee36e8b5c0fdd6014a0398aed18ce9876360bd63 # tag: v3.5.1
+    - name: Set up Bun
+      uses: oven-sh/setup-bun@a1800f471a0bc25cddac36bb13e6f436ddf341d7 # v1
       with:
-        node-version-file: ./frontend/.nvmrc
-        cache: 'yarn'
-        cache-dependency-path: '**/yarn.lock'
-    - name: Install yarn dependencies
+        bun-version: latest
```

Comment:
We want to run the same version we're using in our app here - simply for consistency; so in this case it'd be `489fa7b8b7a2f1dd58968bdf5b360fa144ac1d30bdcfdd98cdda33e33e6fbf22`